src/shared/App.js

```diff
@@ -35,11 +39,13 @@ function App() {
   if (loading) return 'Loading...';
   if (error) return `Error! ${error.message}`;
 
+  dispatch(currentUser(client)); // Fetch user data for Redux (if logged in)
+
   return (
     <Router>
       <React.Fragment>
         <Switch>
-          <Route exact path="/" component={Home} />
+          <Route exact path="/" component={ProtectedRoute} />
```

Comment:
I think it's out of scope for this PR, but we probably want to add a loading state for the `getActiveUser` and `Login` API calls (that I guess we can render conditionally in `ProtectedRoute` or `App`

Just something to keep in mind for the future

Reply:
Yeah for sure! I was also wondering why we aren't using the userQuery and useMutation hooks to make our API calls to the backend? The [result of these hooks](https://www.apollographql.com/docs/react/data/mutations/#result) automatically tells us whether the call is loading/errored/done, so maybe it's an option to consider?

Reply:
yeah, the API of those hooks are super nice — we can look into this in a separate PR

I wonder if it has to do with having to put the result of these in Redux state or something which is why those hooks aren't used here but are used in other places in the app